rust/repositories.bzl

```diff
@@ -83,6 +83,18 @@ def rust_repositories(
         sha256s = sha256s,
     )
 
+    rust_repository_set(
+        name = "rust_linux_aarch64",
+        exec_triple = "aarch64-unknown-linux-gnu",
+        extra_target_triples = ["wasm32-unknown-unknown"],
+        version = version,
+        iso_date = iso_date,
+        rustfmt_version = rustfmt_version,
+        edition = edition,
+        dev_components = dev_components,
+        sha256s = sha256s,
+    )
+
```

Comment:
Thanks!  Generally good with everything in the PR, but a little hesitant to have this function now pull 4 toolchains when most folks need 1 of these, sometime 2.  I'd generally encourage calling rust rust_repository_set explicitly.

Is there a pattern of usage that you're seeing?

Perhaps this function could become more generally useful if the exec_triple's called were a parameter?

```bazel
# maintain backwards compatibility
if exec_triples is None:
    exec_triples = [
        "rust_linux_x86_64",
        "rust_darwin_x86_64,
        "rust_freebsd_x86_64",
        "rust_windows_x86_64",
    ]

for exec_triple in exec_triples:
    rust_repository_set(
        name = "rust_linux_aarch64",
        exec_triple = exec_triple,
        extra_target_triples = ["wasm32-unknown-unknown"],
        version = version,
        iso_date = iso_date,
        rustfmt_version = rustfmt_version,
        edition = edition,
        dev_components = dev_components,
        sha256s = sha256s,
    )
```

Reply:
Aren't the toolchains lazily fetched?

Reply:
Ah, yeah.  Good point.